tests/python_tests/samples/conftest.py

```
@@ -0,0 +1,160 @@
+}
+
+TEST_FILES = {
+    "how_are_you_doing_today.wav": "https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/librispeech_s5/how_are_you_doing_today.wav",
+    "adapter_model.safetensors": "https://huggingface.co/smangrul/tinyllama_lora_sql/resolve/main/adapter_model.safetensors"
+}
+
+SAMPLES_PY_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../samples/python"))
+SAMPLES_CPP_DIR = os.environ.get("SAMPLES_CPP_DIR", os.getcwd())
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--cleanup-cache", action="store_true", default=False, help="Cleanup cache after tests"
+    )
+
+@pytest.fixture(scope="session", autouse=True)
+def setup_and_teardown(request, tmp_path_factory):
+    """Fixture to set up and tear down the temporary directories."""
+    
+    ov_cache = os.environ.get("OV_CACHE", tmp_path_factory.mktemp("ov_cache"))
+    models_dir = os.path.join(ov_cache, "test_models")
+    test_data = os.path.join(ov_cache, "test_data")
+    
+    logger.info(f"Creating directories: {models_dir} and {test_data}")
+    os.makedirs(models_dir, exist_ok=True)
+    os.makedirs(test_data, exist_ok=True)
+    
+    request.config.cache.set("OV_CACHE", str(ov_cache))
+    request.config.cache.set("MODELS_DIR", str(models_dir))
+    request.config.cache.set("TEST_DATA", str(test_data))
+    
+    yield
+    
+    if request.config.getoption("--cleanup-cache"):
+        if os.path.exists(ov_cache):
+            logger.info(f"Removing temporary directory: {ov_cache}")
+            shutil.rmtree(ov_cache)
+        else:
+            logger.info(f"Skipping cleanup of temporary directory: {ov_cache}")
+
+@pytest.fixture(scope="session")
+def convert_model(request):
+    """Fixture to convert the model once for the session."""
+    models_cache = request.config.cache.get("MODELS_DIR", None)
+    model_id = request.param
+    if model_id not in MODELS:
+        raise ValueError(f"Model ID '{model_id}' not found in MODELS dictionary.")
```

```suggestion
```
Just let it fail. The default exception would print something similar

---

done